j/jekyll-commonmark/jekyll-commonmark_ubi_8.5.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : jekyll-commonmark
+# Version       : v1.2.0
+# Source repo   : https://github.com/pathawks/jekyll-commonmark
+# Tested on     : UBI: 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: MIT License
```

Comment:
The Script license should always be Apache 2.0, note that it is not the same as package license